packages/material-ui/src/GridListTile/GridListTile.js

```diff
@@ -65,7 +65,29 @@ function ensureImageCover(imgEl, classes) {
   }
 }
 
+let warnedOnce = false;
+
+/**
+ * ⚠️ The GridList component was renamed to imageList to align with the current Material Design naming.
+ *
+ * You should use `import { ImageListTile } from '@material-ui/core'`
+ * or `import ImageListTile from '@material-ui/core/ImageListTile'`.
+ */
 const GridListTile = React.forwardRef(function GridListTile(props, ref) {
+  if (process.env.NODE_ENV !== 'production') {
+    if (!warnedOnce) {
+      warnedOnce = true;
+      console.error(
+        [
+          'Material-UI: The GridListTile component was renamed to ImageListTile to align with the current Material Design naming.',
```

Comment:
Looking at the spec, this is referred to as an "image list item". Should we do the same? (Wishing I'd check that before completing the rename on, and back-porting, but happy to do a bit of extra work if it makes sense.)

Reply:
No strong preference

Reply:
In that case I think ImageListItem makes most sense, so that it's consistent with both the spec and List:

- List, ListItem
- ImageList, ImageListItem

Then, what to call GridListTileBar? The Card has CardHeader, so ImageListItemFooter? 🤷‍♂️

Reply:
The reason I don't have a strong point of view is because the current names are more descriptive. Tile > Item. Bar can be a header or a footer. But at the same time a generic name can be easier to remember.

Reply:
Meet you half way? (Change Tile to Item, but keep Bar?)

Reply:
Sounds great